docs/document/content/others/api-change-history/shardingsphere-jdbc/spring-boot-starter.en.md

```diff
@@ -3,7 +3,7 @@ title = "Change History"
 weight = 7
 +++
 
-## 5.0.0-alpha
+## 5.0.0-beta
 
 ### Replica Query
```

Comment:
in 5.0.0, this feature named [READWRITE-SPLITTING](
https://shardingsphere.apache.org/document/current/en/user-manual/shardingsphere-jdbc/configuration/spring-boot-starter/readwrite-splitting/), please change the name, thank you.